### PR TITLE
Fixes #27858 - Add bulk resume ui to tasks

### PR DESCRIPTION
**Pull request by MariaAga**
Select the tasks you want to resume/cancel by using select boxes in the table and choose the action with the 'select action' button drop down
If one of the selected tasks is not cancelable/resumable that task will not be attempted to be canceled/resumed.
Select all button will select only the tasks in the page

### CODE REVIEW

File: webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js [hunk 1]
```diff
@@ -93,18 +95,25 @@ export const unselectRow = id => ({
...
-    notAllCancelleble = true;
+    if (actionType === RESUME && task.isResumeble)
+      return dispatch(resumeTask(task.id));
+    else if (actionType === CANCEL && task.isCancelleble)
+      return dispatch(cancelTask(task.id));
+    notAllActionable = true;
     return null;
   });
-  if (notAllCancelleble)
+  if (notAllActionable)
```
Comment (amirfefer) [inline webpack/ForemanTasks/Components/TasksTable/TasksTableActions.js]: tasks table reloads even though none of the tasks are resumable/cancelable and no data has been changed

**Comment by adamruzicka** [comment]
Needs rebase

**Comment by MariaAga** [comment]
@amirfefer Added the modal for resume action as well,
@adamruzicka The confirm text for resume says "This will resume 3 tasks, putting them in the running state.  Are you sure?" is this correct?

**Comment by adamruzicka** [comment]
Yes, that sounds right

**Comment by MariaAga** [comment]
@amirfefer I dont think it makes sense to make the button a drop down because a task can't be cancelable and resumable at the same time. (@adamruzicka  am correct?)

**Comment by MariaAga** [comment]
Cancel confirmation will be added it a separate PR.
Title actions will be added in https://github.com/theforeman/foreman-tasks/pull/462

**Comment by adamruzicka** [comment]
Task can be either resumable or cancelable, but not both at the same time. Needs a rebase

**Comment by MariaAga** [comment]
rebased

**Comment by MariaAga** [comment]
@adamruzicka 
1. Would that be from the server side?
2. What if you select 20 tasks but only 10 are canceled?

**Comment by adamruzicka** [comment]
1) The server would receive a search query and remove everything that matches.
2) I'm afraid I don't have an answer to that yet, maybe we could bring the UXD team to this

**Comment by adamruzicka** [comment]
Needs a rebase now that #463 was merged

**Comment by adamruzicka** [comment]
@MariaAga could you please fix travis?

**Pull request merged**